corehq/apps/users/util.py

```diff
@@ -545,7 +545,7 @@ def percentage_complete(self):
         if self.total and self.current:
             if self.total == 0:
                 return None
-            if self.total > self.current:
+            if self.total < self.current:
```

Comment:
Will `self.current` ever be greater than `self.total`? If there are 5 users to clear, and all 5 are cleared, then I imagine `self.total == self.current`?

Reply:
Correct, current should (and can theoretically) never be greater than total, but I modelled this after the `MessagingRuleProgressHelper` which does it that way (maybe for a good reason?) and thought it's probably more of a good thing than a bad thing.